### ISSUE TITLE
Base map labels not visible when adding Satellite Imagery 

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/terriajs#3206]
When users load satellite layers, the land markings disappear e.g. place names, state boundaries and reference points - makes it difficult to orientate yourself, when you reduce the transparency you can no longer see the layer. Can the geographic information be layered over the top? I don't think the contrasts between map layers and geographic layers would meet accessibility requirements and would be very difficult for some people to view

https://map.drought.gov.au/#share=s-7sRHFfbTLsFBGu2Si7pAR5SKKgK

**Comment by AnaBelgun** [comment]
@philipgrimmett it would be great to discuss this

**Comment by meh9** [comment]
I added the Place Names layer to the catalogue to try to alleviate this, but it's at the bottom of the catalogue, not on by default, and it's not a great layer.

Other maps have asked for this as well, but I've not been able to find a good layer that shows things like that.

Maybe it's time we start hosting out own layer for this?

**Comment by philipgrimmett** [comment]
@meh9 are we talking about an always on, alway on top layer? Visible in the workbench or hidden?

**Comment by meh9** [comment]
@philipgrimmett yeah I think always on and always on top, with special UI elements for it:

<img width="663" alt="mock" src="https://user-images.githubusercontent.com/5283003/53166534-a87f8800-3629-11e9-9f69-632be7fa7a12.png">

**Comment by hardreddata** [comment]
As an end user (and hopeful skinned TerriaMap maintainer) I just assumed we had to do this ourselves, taking advantage of web services where possible.

In NSW there is https://maps.six.nsw.gov.au/arcgis/rest/services/sixmaps/LPI_RasterLabels_1/MapServer and so when combined with their public imagery we get https://map.drought.gov.au/#share=s-wxE0tVQcl18bSmiwK13Ca6P1qB3

Stamen make this quite available through their _label only_ url templates http://maps.stamen.com/terrain-labels/#16/-33.9129/151.2060 

It looks like Carto may also do this through their style argument https://github.com/CartoDB/basemap-styles though I haven't tested it. 

That said I would love a national layer for this, along the lines of a hosted, current, Geodata Topo. https://ecat.ga.gov.au/geonetwork/srv/eng/catalog.search?node=srv#/metadata/64058

I hope this helps.
